pkg/client/auth/kerberos/spnego_unix.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux || darwin
```

Comment:
Is this needed here?

Reply:
The goimports linter keeps complaining if it isn't there. I don't believe it's necessary though

Reply:
Don't see it reported as a linter error, I'd leave it out of this PR

Reply:
Removed